### PR TITLE
Fix symlink logic in package install

### PR DESCRIPTION
**Pull request by mamckee**
Post-install script attempted to create existing symlink created by package install util. Removed this command and fixed package cleanup

### CODE REVIEW

File: test/functional/src/ec_tests.cpp [hunk 1]
```diff
@@ -29,6 +35,11 @@ bool EcTests::IsConfigured()
         return TestFail("keysinuse engine not loaded for EC operations, found [ %s ]", loaded_id);
     }
 
+    return true;
+}
+
+bool EcTests::Setup()
+{
     if (ecBio == nullptr)
```
Comment (mlindgren) [inline test/functional/src/ec_tests.cpp:45]: >if (ecBio == nullptr) [](http://example.com/codeflow?start=4&length=21)

Where does this get set? #Resolved

File: test/functional/src/main.cpp [hunk 2]
```diff
@@ -98,12 +111,11 @@ int main(int argc, char **argv)
...
 
     // RSA APIs
+    if (isRsaConfigured)
     {
-        bool isConfigured;
         RsaTests rsaTests(logLocation);
-        RunTest("== RSA Configuration ==", [&rsaTests, &isConfigured](){return (isConfigured = rsaTests.IsConfigured());});
 
-        if (isConfigured)
+        if (RunTest("== RSA setup ==",            [&rsaTests](){return rsaTests.Setup();}))
```
Comment (mlindgren) [inline test/functional/src/main.cpp]: >[&rsaTests](){return rsaTests.Setup();})) [](http://example.com/codeflow?start=50&length=41)

Nit: I think you can do this without a lambda by passing in `std::bind(&RsaTests::Setup, rsaTests)` #Resolved

File: test/functional/src/ec_tests.cpp (hunk 1 above)
Comment (mamckee) [reply test/functional/src/ec_tests.cpp:45]: Constructor. I'll move that logic here though.

Comment (mlindgren) [reply test/functional/src/ec_tests.cpp:45]: Ah, right. Missed that. I do think it makes sense to initialize it and check the result in the same place.

**Review by mlindgren — APPROVED** [review]
:shipit:

**Pull request merged**